reo/nested_inputs.py

```diff
@@ -1183,6 +1386,470 @@ def list_of_int(input):
           "default": False,
           "description": "True/False for if technology can curtail energy produced."
         }
+      },
+      "CHP": {
+        "prime_mover": {
+          "type": "str",
+          "description": "CHP prime mover type (recip_engine, micro_turbine, combustion_turbine, fuel_cell)"
+        },
+        "size_class": {
+          "type": "int",
+          "restrict_to": [0, 1, 2, 3, 4, 5, 6],
+          "description": "CHP size class for using appropriate default inputs"
+        },
+        "min_kw": {
+          "type": "float",
+          "min": 0.0,
+          "max": 1.0e9,
+          "description": "Minimum CHP size (based on electric) constraint for optimization"
+        },
+        "max_kw": {
+          "type": "float",
+          "min": 0.0,
+          "max": 1.0e9,
+          "description": "Maximum CHP size (based on electric) constraint for optimization. Set to zero to disable CHP"
+        },
+        "min_allowable_kw": {
+          "type": "float",
+          "min": 0.0,
+          "max": 1.0e9,
+          "description": "Minimum CHP size (based on electric) that still allows the model to choose zero (e.g. no CHP system)"
+        },
+        "installed_cost_us_dollars_per_kw": {
+          "type": ["float", "list_of_float"],
+          "min": 0.0,
+          "max": 1.0e5,
```

Comment:
@Bill-Becker is there a reason this is not 1e9?

Reply:
I just used this number because that's already unrealistically high for this cost.